### PR TITLE
Fix #121

### LINKED ISSUE #121: Add loading spinners on classroom overview stats

### PR DESCRIPTION
**Pull request by simoneduca**

### CODE REVIEW

**Comment by simoneduca** [comment]
@eatyourgreens or @shaunanoordin should be a quick one, thanks.

File: src/presentational/ClassroomsOverview.jsx [hunk 1]
```diff
@@ -16,7 +18,9 @@ const ClassroomsOverview = (props, context) => (
                 <i className="fa fa-institution fa-4x"></i>
               </div>
               <div className="col-xs-9 text-right">
+                {(context.classrooms.data.length > 0) ?
                 <h1>{context.classrooms.data.length}</h1>
```
Comment (shaunanoordin) [inline src/presentational/ClassroomsOverview.jsx:22]: @simoneduca , I need to check: if a new user accesses this page for the first time, won't `context.classrooms.data.length` and `context.classrooms.members.length` both be 0? If yes, wouldn't new users always see the spinners, until they create a class and have a student?

Is there another way to check if the values have been properly loaded or not, perhaps `context.classrooms.loading === false`?

Comment (simoneduca) [inline src/presentational/ClassroomsOverview.jsx:22]: Derp, you're right! Fancy adding the fix?

Comment (shaunanoordin) [inline src/presentational/ClassroomsOverview.jsx:22]: Sure can do! Give me 5 mins.

**Comment by shaunanoordin** [comment]
Awright, we're updated! :) I'm happy to continue merging if you have no issues, @simoneduca

**Comment by shaunanoordin** [comment]
Status: **Works!**
Actions: **Merging**

Tested with a virgin Zoo account (no classrooms, no students) as well as a Zoo account that has classrooms and students, on Chrome49+MacbookPro+OSX.

**Pull request merged**